ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -76,7 +76,7 @@ public void Export(Resource resource, string targetPath)
         /// <param name="resourceName">The name of the resource being exported.</param>
         /// <param name="fileName">Target file name.</param>
         /// <param name="world">The world resource to export.</param>
-        public void ExportToFile(string resourceName, string fileName, VWorld world)
+        public void ExportToFile(string resourceName, string fileName, VWorld world, CancellationToken? cancellationToken)
```

Comment:
Need to add an xmldoc.

Seems like `public void Export` is missing an xmldoc entirely too.

Reply:
Done.